ts/screens/onboarding/TosScreen.tsx

```diff
@@ -62,22 +62,24 @@ const TosScreen: React.SFC<Props> = ({
           : I18n.t("onboarding.tos.headerTitle")
       }
     >
-      <Content>
-        {isAcceptingTos && (
-          <View style={styles.activityIndicatorContainer}>
-            <ActivityIndicator />
-          </View>
-        )}
-        {/* FIXME: handle errors */}
-        {profileUpsertError.isSome() && (
-          <View padder={true}>
-            <Text>{I18n.t("global.actions.retry")}</Text>
-          </View>
-        )}
-        <Markdown lazyOptions={{ lazy: true }}>
-          {I18n.t("profile.main.privacy.text")}
-        </Markdown>
-      </Content>
+      <ScrollView>
+        <Content scrollEnabled={false}>
```

Comment:
mmh not sure about this, [content is already a scrollview](https://github.com/GeekyAnts/NativeBase/blob/master/src/basic/Content.js#L91)